src/Microsoft.ApiDesignGuidelines.Analyzers/Core/DoNotDirectlyAwaitATask.Fixer.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace Microsoft.ApiDesignGuidelines.Analyzers
```

Comment:
Is this something we want generally? We do this in Roslyn so we can allow people to safely .Result in various cases, but I'm not sure I'd want to pass this off as a universally applicable rule.